server/src/modules/clubs/domain/entities/club.ts

```diff
@@ -7,6 +7,22 @@ import { ClubId } from '../value-objects/clubId'
 interface ClubProps {
   name: string
   description: string
+  iconImage: string
+  bannerImage: string
+  facebook?: string
+  twitter?: string
+  instagram?: string
+  website?: string
```

Comment:
I feel like the domain type should change too? All these fields being under `links`?

Reply:
changed in latest pr